sql-db/vertx-sql/src/test/java/io/quarkus/ts/vertx/sql/dbpool/PostgresqlResource.java

```diff
@@ -20,6 +20,9 @@ public Map<String, String> start() {
                 .withEnv("POSTGRES_USER", "test")
                 .withEnv("POSTGRES_PASSWORD", "test")
                 .withEnv("POSTGRES_DB", "amadeus")
+                .withEnv("POSTGRESQL_USER", "test")
```

Comment:
I'd expect to have some comment, but as it is hotfix for 3.8 and not main, I don't mind.

Reply:
Added it. Also sorry for forget put branch name in title